addons/web_editor/static/src/components/media_dialog/media_dialog.scss

```diff
@@ -1,114 +1,55 @@
-$min-row-height: 128;
-
 .modal:not(.o_legacy_dialog) .o_select_media_dialog {
-    .nav-item {
-        cursor: pointer;
-    }
-
-    .o_we_horizontal_collapse {
-        width: 0 !important;
-        padding: 0 !important;
-        border: none !important;
-    }
-
-    .o_we_transition_ease {
-        transition: all ease 0.25s;
-    }
 
     .o_we_existing_attachments {
-        min-height: $min-row-height + px;
-
-        .o_we_attachment_placeholder {
-            flex-grow: $min-row-height;
-            flex-basis: $min-row-height + px;
-        }
-
         .o_existing_attachment_cell {
-            cursor: pointer;
-            margin: 1px;
-
             &.o_we_image {
                 position: fixed !important;
                 opacity: 0;
-                transition: opacity 0.5s;
-                transition-delay: 0.5s;
+                transition: opacity 0.5s ease 0.5s;
 
                 &.o_loaded {
                     position: relative !important;
                     opacity: 1;
```

Comment:
it should be possible to add both `position-relative opacity-100` classes within `o_loaded`, could you check?